bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java

```diff
@@ -63,12 +63,13 @@
             + "facet=etat_vent&facet=etat_pluie_inondation&facet=etat_orage&facet=etat_inondation&facet=etat_neige&facet=etat_canicule&"
             + "facet=etat_grand_froid&facet=etat_avalanches&refine.nom_dept=";
     private static final int TIMEOUT_MS = 30000;
+    private static final String UNKNOWN_COLOR = "b3b3b3";
     private static final Map<AlertLevel, String> ALERT_COLORS = Map.ofEntries(
             new AbstractMap.SimpleEntry<AlertLevel, String>(AlertLevel.GREEN, "00ff00"),
             new AbstractMap.SimpleEntry<AlertLevel, String>(AlertLevel.YELLOW, "ffff00"),
             new AbstractMap.SimpleEntry<AlertLevel, String>(AlertLevel.ORANGE, "ff6600"),
             new AbstractMap.SimpleEntry<AlertLevel, String>(AlertLevel.RED, "ff0000"),
-            new AbstractMap.SimpleEntry<AlertLevel, String>(AlertLevel.UNKNOWN, "b3b3b3"));
+            new AbstractMap.SimpleEntry<AlertLevel, String>(AlertLevel.UNKNOWN, UNKNOWN_COLOR));
```

Comment:
It would be so much easier to read when statically importing `Map.entry` and using `entry(` instead of all those lengthy `new AbstractMap.SimpleEntry<AlertLevel, String>(` constructs here. :laughing:

```java
private static final Map<AlertLevel, String> ALERT_COLORS = Map.ofEntries(
            entry(AlertLevel.GREEN, "00ff00"),
            entry(AlertLevel.YELLOW, "ffff00"),
            entry(AlertLevel.ORANGE, "ff6600"),
            entry(AlertLevel.RED, "ff0000"),
            entry(AlertLevel.UNKNOWN, UNKNOWN_COLOR));
```

Reply:
Since we will not backport to 2.5.x we could also use `Map.of`. But I think changing all this is beyond the scope here.

Reply:
I just hope a casual reader learns something from it so there will be less of these monstrosities in the future. :-)